tests/test_turbot.py

```diff
@@ -66,12 +109,14 @@ def someone():
 
 def someturbotadmin():
     """Returns a random non-admin user with the Turbot Admin role"""
-    return random.choice([BUDDY, DUDE])
+    cond = lambda member: member != ADMIN and ADMIN_ROLE in member.roles
+    return random.choice(list(filter(cond, CHANNEL_MEMBERS)))
 
 
 def somenonturbotadmin():
     """Returns a random non-admin user without the Turbot Admin role"""
-    return random.choice([FRIEND, GUY])
+    cond = lambda member: member != ADMIN and ADMIN_ROLE not in member.roles
+    return random.choice(list(filter(cond, CHANNEL_MEMBERS)))
```

Comment:
Made these changes just so these functions with _just work_ if we happen to mess with which example user has which roles, and/or we add/delete more users.

Reply:
Will freely admit I got a bit lazy there